pkg/cmd/cli/cmd/newapp.go

```diff
@@ -66,6 +66,9 @@ You can use '%[1]s status' to check the progress.`
   # List all local templates and image streams that can be used to create an app
   %[1]s new-app --list
 
+  # Create an application based on environment file provided.
+  %[1]s new-app openshift/postgresql-92-centos7 --env-file=env-file-location
```

Comment:
Not entirely correct. The application is being created *based* on `openshift/postgresql-92-centos7`, not the env file. I'd make the source arg simpler to focus on what's being exampled here (the env var file), and also make it clearer that it takes the path to local file. Something like:

```
# Use the public Docker Hub MySQL image to create an app. On each container, set multiple environment variables present in a file containing key-value pairs
%[1]s new-app mysql --env-file=/path/to/env-file
```

And put it after the other MySQL example.